Client/src/Components/Navigation/ListRouter.tsx

```diff
@@ -10,7 +10,7 @@ import CloudUploadIcon from '@material-ui/icons/CloudUpload'
 import { DatasetView } from "../DatasetUpload/DatasetView"
 import DonutSmallIcon from '@material-ui/icons/DonutSmall'
 import ForgotPasswordView from "../Authentication/ForgotPasswordView"
-import GraphView from "../Graph/GraphView"
+import { GraphView } from "../Graph/GraphView"
```

Comment:
This seems like an unneeded change? Not that it matters

Reply:
{ } mean that a component that is being imported is not _default_ exported from the file, it's "just" exported. I dislike default export a little, primarily because it allows the importer to name the imported component in whatever name they want
https://medium.com/@etherealm/named-export-vs-default-export-in-es6-affb483a0910